css/properties.json

```diff
@@ -6948,7 +6948,7 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/outline-offset"
   },
   "outline-style": {
-    "syntax": "auto | <'border-style'>",
+    "syntax": "auto | <'outline-line-style'>",
```

Comment:
We also need to have outline-line-style defined somewhere.

Reply:
Hey @teoli2003 - for getting this resolved, I'm not finding any MDN URL specified for this. I'm going to head over to the mdn/content repo and submit a PR for this. 

Is this what we had in mind?

Reply:
Or was it something along the lines of a similiar example, [outline-width](https://developer.mozilla.org/en-US/docs/Web/CSS/outline-width#line-width)

Reply:
I think we should list the valid values and not use  <'outline-line-style'>". It seems not useful to create a whole type page for something used in one place.

Reply:
@timeowilliams Do you still want to add the values?

Basically ([source](https://www.w3.org/TR/css-ui-4/#typedef-outline-line-style)):

> `<outline-line-style> accepts the same values as <line-style> (CSS Backgrounds 3 §3.2 Line Patterns: the border-style properties) with the same meaning, except that hidden is not a legal outline style. In addition, the outline-style property accepts the value auto.`

And `line-style` accepts these values ([source](https://www.w3.org/TR/css-backgrounds-3/#typedef-line-style)):

> `<line-style> = none | hidden | dotted | dashed | solid | double | groove | ridge | inset | outset`